README.md

```diff
@@ -11,6 +11,7 @@ standalone script or during bootup via a dracut module.
 * `coreos.inst.image_url` - The URL of the CoreOS image to install to this device
 * `coreos.inst.ignition_url` - The URL of the CoreOS Ignition config (optional, enter
   `coreos.inst.ignition_url=skip` to not load an Ignition config)
+* `coreos.inst.platform_id` - The Ignition platform ID the CoreOS image to install to
```

Comment:
`the CoreOS image` doesn't really seem to fit into this description in the current wording. Maybe `The Ignition platform ID the CoreOS image is being installed on`

Reply:
or `The Ignition platform ID of the platform the CoreOS image is being installed on`

Reply:
Thanks for the suggestion! Updating..